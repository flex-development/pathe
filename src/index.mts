/**
 * @file Package Entry Point
 * @module pathe
 */

export type * from '#interfaces/index'
export * from '#lib'
export { default, posix, win32 } from '#pathe'
export type * from '#types/index'
