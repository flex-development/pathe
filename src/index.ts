/**
 * @file Package Entry Point
 * @module pathe
 */

export * from './lib'
export * from './types'
