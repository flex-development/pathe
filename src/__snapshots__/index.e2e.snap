// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`e2e:pathe > should expose public api 1`] = `
[
  "addExt",
  "basename",
  "changeExt",
  "cwd",
  "delimiter",
  "dirname",
  "dot",
  "extname",
  "extnames",
  "format",
  "formatExt",
  "isAbsolute",
  "isDeviceRoot",
  "isSep",
  "join",
  "matchesGlob",
  "normalize",
  "parse",
  "relative",
  "removeExt",
  "resolve",
  "resolveWith",
  "root",
  "sep",
  "toNamespacedPath",
  "toPosix",
  "default",
  "posix",
  "win32",
]
`;
